packages/manager/apps/catalog-revamp/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,205 @@
+                ) : (
+                  <OsdsText
+                    color={ODS_THEME_COLOR_INTENT.default}
+                    level={ODS_TEXT_LEVEL.button}
+                    size={ODS_TEXT_SIZE._600}
+                  >
+                    {label} {resultsNumber}
+                  </OsdsText>
+                )}
+              </span>
+            </OsdsCheckboxButton>
+          </OsdsCheckbox>
+        </div>
+      );
+    };
+
+    return (
+      <>
+        <div className="filters-container filters-container grid grid-cols-1 md:flex">
+          <div className="filters-universes">
+            <OsdsText
+              level={ODS_TEXT_LEVEL.heading}
+              size={ODS_TEXT_SIZE._400}
+              color={ODS_THEME_COLOR_INTENT.text}
+              className="title"
+            >
+              {t('manager_catalog_filters_universes')}
+            </OsdsText>
+            {universes.map(
+              (item: { universe: string; count: number }, id: number) => (
+                <FilterItem
+                  key={`item-${id}`}
+                  label={item.universe}
+                  count={item.count}
+                  type="universe"
+                />
+              ),
+            )}
+            {!universes.length &&
+              [...Array(5)].map((_, id: number) => (
+                <FilterItem key={`item-${id}`} loading={true} type="universe" />
+              ))}
+          </div>
+          <div className="filters-categories grid">
+            <OsdsText
+              level={ODS_TEXT_LEVEL.heading}
+              size={ODS_TEXT_SIZE._400}
+              color={ODS_THEME_COLOR_INTENT.text}
+              className="title"
+            >
+              {t('manager_catalog_filters_categories')}
+            </OsdsText>
+            <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
+              {categories.map(
+                (item: { category: string; count: number }, id: number) => (
+                  <FilterItem
+                    key={`item-${id}`}
+                    label={item.category}
+                    count={item.count}
+                    type="category"
+                  />
+                ),
+              )}
+              {!categories.length &&
+                [...Array(15)].map((_, id: number) => (
+                  <FilterItem
+                    key={`item-${id}`}
+                    loading={true}
+                    type="category"
+                  />
+                ))}
+            </div>
+          </div>
+        </div>
+        <div className="filters-footer-container">
+          <div className="filters-footer flex justify-end gap-x-5">
+            <OsdsLink
+              color={ODS_THEME_COLOR_INTENT.primary}
+              onClick={() => resetFilters()}
+            >
+              {t('manager_catalog_filters_reset')}
+            </OsdsLink>
+            <OsdsButton
+              disabled={!hasInteracted ? true : undefined}
```

Comment:
```suggestion
              disabled={!hasInteracted}
```

It doesn't work like this ?

or `!hasInteracted ?? undefined`

Reply:
No it won't work it's a problem with OsdsButton that I putted in the ODS channel.
disabled have to be `undefined` to work. false will not work.

Reply:
the second solution puts undefined

Reply:
I have tested second solution, unfortunately it's not working

Reply:
`disabled={!hasInteracted ? true : undefined}` works - like it is right now
`disabled={!hasInteracted ?? undefined}` not working

Reply:
What is the difference ?

Reply:
First case : <OsdsButton disabled ...> or <OsdsButton ...> without disabled attribute. Tested locally and working.
Second case : <OsdsButton disabled ...> or <OsdsButton disabled={false} ...>. Tested locally and not working.
You are proposing the Nullish Coalescing Operator (??) and the behavior is not the same.
You can test it locally and you will see that this solution which seems good is unfortunately not working in this context.

Reply:
This is working. Done.